server/views/pages/overview/partials/timelineTab/_timelineEvent-INDUCTION_UPDATED.njk

```diff
@@ -0,0 +1,10 @@
+<div class="moj-timeline__item" data-qa-event-type="INDUCTION_UPDATED">
+  <div class="moj-timeline__header">
+    <h2 class="moj-timeline__title">{{ 'INDUCTION_UPDATED' | formatTimelineEvent }}</h2>
```

Comment:
Very minor thing but it seems the filter isn't supplying much value anymore. Maybe we can just inline the text and remove the filter, unless it's used elsewhere and we want to keep the content consistent in which case it's fine?